luda-editor/luda-editor-client/src/app/sequence_list/mod.rs

```diff
@@ -93,15 +112,38 @@ impl Entity for SequenceList {
                 SequenceListEvent::ScrolledEvent { scroll_y } => {
                     self.scroll_y = *scroll_y;
                 }
+                SequenceListEvent::PreviewSliderMovedEvent { path, progress } => {
+                    if let Some(load_state) = self.sequence_load_state_map.get(path) {
+                        if let SequenceLoadStateDetail::Loaded { sequence } = &load_state.detail {
+                            let duration = calculate_sequence_duration(
+                                sequence,
+                                &self.subtitle_play_duration_measurer,
+                            );
+                            let moved_time = duration * progress;
+                            self.sequence_player.update_sequence(sequence.clone());
```

Comment:
`SequencePlayer::update_sequence()` will be called even if sequence didn't changed.
I think this can lead to poor performance.
Do you think I need to fix it now?

Reply:
If you face the performance issue, yes.
If not, no.
Did you get any trouble with performance issue by multiple PreviewSliderMovedEvent?

Reply:
No, there is no problem in my case